lib/diff.ml

```diff
@@ -429,11 +429,13 @@ and signatures ~reference ~current =
     items ~reference:modified_reference ~current:modified_current ~typing_env
   with
   | [] -> (
+      Compmisc.init_path ();
+      let initialized_env = Compmisc.initial_env () in
```

Comment:
Might be worth extracting a function to build a fresh environment with the stdlib (and later, any external dependencies) loaded.

I think `Typing_env` would be a good place to write this function, especially since it will use it internally as well!